docker/docker-compose.yaml

```diff
@@ -35,6 +35,10 @@ services:
   cmake:
     <<: *common
     command: /bin/bash -xcl "cmake -G Ninja -D CMAKE_BUILD_TYPE=Release -B out -S . && ninja -C out"
+    
+  cxx-interop-build:
```

Comment:
Need to add `cxx-interop-build` to `docker-compose.2204.59.yaml` etc. to override image.